cli/src/main/resources/help/command.txt

```diff
@@ -91,3 +100,18 @@ ARGUMENTS
             removed.
 			
   list    - lists existing generic type commands.
+ 
+
+  For example,Login to CLI mode and run the below commands,
```

Comment:
Perhaps an EXAMPLE chapter containing the list/add/remove examples? Do we need "Login to CLI mode and run"?